### PR TITLE
PR for Smashing Magazine Heatmap (feature branch)

### PR DESCRIPTION
**Pull request by UmairAhmad125**
This PR is for Design Teardown (Smashing Magazine's heatmap).
The project includes a heatmap of Smashing magazine using shades of Gray color.
The project was manually tested.
Details of the project are in the readme file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,626 @@
+<!DOCTYPE html>
```
Comment (lucasmazo32) [inline index.html:1]: - [x] Kindly fix the HTML file so you don't have any warnings.

> Warnings
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,626 @@
...
+        </aside>
+        <footer class="footer">
+            <div class="ftitle">
+                <h2>BROWSE ALL SMASHING MAGAZINE TOPICS</h2>
+            </div>
+            <div class="foot-content">
+                <p>
+                    Accessibility Android Animation Apps CSS Design Design Patterns Design Systems E-Commerce Freebies Graphics HTML Illustrator Inspiration iOS Java Script Mobile Pattern Libraries Performance Photoshop Plugins React Responsive Web Design Service Workers
+                    Sketch Typography UI Usability User Experience Wallpapers Web Design WordPress Workflow
+                </p>
```
Comment (lucasmazo32) [inline index.html]: - [x] This items should be links. Kindly fix that.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,558 @@
...
+  background-color: rgb(59, 59, 59);
+  border-radius: 4px;
+  transform: rotate(-20deg);
+  margin-top: 20px;
+  box-shadow: 0 10px rgba(46, 41, 41, 0.178);
+}
+
+.imgTitle:hover .img {
+  transform: rotate(360deg);
+}
```
Comment (lucasmazo32) [inline css/style.css]: - [OPTIONAL] First of all, kudos on making the rotation! I will have advice, so this animation will look better and smooth.

Add these lines to `.img`:
```
transform-origin: 0 100%;
transition: all 0.5s ease;
```

The changes mentioned before will help you modify the desired axis.

Also, change `.imgTitle:hover .img` to the following:
```
.img:hover {
  transform: rotate(0);
}
```

Now the animation will only trigger when you hover over the image.

**Review by lucasmazo32 — CHANGES_REQUESTED** [review]
# Changes required :wrench:

You are doing a great job so far, and the project is looking good. 

For the project to get approved, kindly implement the changes mentioned below.

### Happy coding! :computer:

## Lucas Mazo

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

File: css/style.css (hunk 3 above)
Comment (addod19) [reply css/style.css]: :100:

File: index.html (hunk 2 above)
Comment (addod19) [reply index.html]: :100:

File: index.html (hunk 1 above)
Comment (addod19) [reply index.html:1]: :100:

**Review by addod19 — APPROVED** [review]
## Status: Require changes :trophy: :balloon: 

Hi @UmairAhmad125, great job, and a great project :+1: Congratulations on fixing all the comments pointed out by the previous TSE :slightly_smiling_face: I hereby approve your project

Go ahead and merge to master
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**